app/views/users/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>All Users</title>
+</head>
+<body>
+  <h1>All Users</h1>
+  <ul>
+    <% @users.each do |user| %>
+     
+      <li><%= user.name %></li>
+      <li>Number of posts: <%= user.posts_counter %></li>
+      
+    <% end %>
+  </ul>
+</body>
+</html>
```

Comment:
- [ ] Per the requirement we should only have placeholder texts. Kindly add only placeholder text as required.